### DISCUSSION TITLE
If elasticsearch queries returned `timed_out` it should give the user a some sort of warning.

### DISCUSSION THREAD

**Discussion by kanapuli** [grafana/grafana#58401] [Data Sources]
### What Grafana version are you using?
Version 4.6.3 (commit: 7a06a47)
### What datasource are you using?
Elasticsearch 5.4.2
### What OS are you running grafana on?
Ubuntu 16.04
### What did you do?
Queried elasticsearch to return 4 billion records which had a response that says timedout = true. so elastic didn't process all the records and returned the response in the middle. 
### What was the expected result?
The expectation is that the grafana dashboard should have prompted the user that the response have timed out and the graph is inconsistent
### What happened instead?
Grafana rendered the graph with the results returned from the response ignoring the elastic timeout property in the response json.

**Comment by aocenas** [comment]
@grafana/grafana-frontend-platform Is showing warnings in dashboard something on your backlog already or is it possible right now. As far as I know something like that is also needed for Graphite in some cases.

**Comment by ifrost** [comment]
@Elfo404, it's quite an old issue, does it still apply to Elastic or is it more general problem for all data sources?

**Comment by Elfo404** [comment]
@ifrost timeout here refers to a soft timeout sent by elasticsearch in the response, not an actual request timeout. it is set on a per-query basis in the response when during multi searches in `responses[n].timed_out`. when set to true, the returned data may contain partial results

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
